ci/infra/testrunner/vars.yaml

```diff
@@ -37,11 +37,11 @@ lb:
   cpu: 4
 
 master:
-  count: 3
+  count: 3 # Amount of masters to provision
   memory: 4096
   cpu: 4
 
 worker:
-  count: 3
+  count: 2 # Amount of workers to provision
```

Comment:
I think we shouldn't change the default configuration just to match the expectations of one test.